requirements/base.Makefile

```diff
@@ -45,7 +45,11 @@ help: ## this colorful help
 #
 %.txt: %.in
 	cd ..; \
-	pip-compile $(UPGRADE_OPTION) --build-isolation --strip-extras --output-file requirements/$@ requirements/$<
+	pip-compile $(UPGRADE_OPTION) \
+		--build-isolation \
+		--strip-extras \
+		--resolver=backtracking \
```

Comment:
mmhh.. interesting, i thought we had thisone in place. Can you please check the history of this file in case we remove it and if so firgure out why?

Reply:
Ahh sorry, different repo 

https://github.com/ITISFoundation/osparc-simcore/blob/37109a9d14f7dba2e0e98967dc40a201c87ad5a6/requirements/base.Makefile#L46-L52